charts/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,319 @@
+# OpenTelemetry Operator Helm Chart
+
+The Helm chart installs [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator) in Kubernetes cluster.
+The OpenTelemetry Operator is an implementation of a [Kubernetes Operator](https://www.openshift.com/learn/topics/operators).
+At this point, it has [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) as the only managed component.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.0+
+
+### TLS Certificate Requirement
+
+In Kubernetes, in order for the API server to communicate with the webhook component, the webhook requires a TLS
+certificate that the API server is configured to trust. There are three ways for you to generate the required TLS certificate.
+
+  - The easiest and default method is to install the cert-manager and keeps `admissionWebhooks.certManager.enabled` to `true`.
+    In this way, cert-manager will generate a self-signed certificate. _See [cert-manager installation](https://cert-manager.io/docs/installation/kubernetes/) for the instruction._
+  - You can also provide your own Issuer by configuring the `admissionWebhooks.certManager.issuerRef` value. You will need
+    to specify the `kind` (Issuer or ClusterIssuer) and the `name`. Note that this method also requires the installation of cert-manager.
+  - The last way is to manually modify the secret where the TLS certificate is stored. You can either do this before installation
+    or after.
+    - To do this before installation, you don't have to install cert-manager. Please set `admissionWebhooks.certManager.enabled` to `false`.
+      - Create the namespace for the OTEL Operator and the secret
+        ```console
+        $ kubectl create namespace opentelemetry-operator-system
+        ```
+      - Config the TLS certificate
+        ```console
+        $ kubectl apply -f - <<EOF
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: opentelemetry-operator-controller-manager-service-cert
+            namespace: opentelemetry-operator-system
+          type: kubernetes.io/tls
+          data:
+            tls.crt: |
+                # your signed cert
+            tls.key: |
+                # your private key
+          EOF
+        ```
+        You can also do this using `kubectl create` command.
```

Comment:
This is the preferred option. I'd recommend listing this first. Or even removing the one above.

Reply:
Yeah, sure. I will make the changes.